README.md

```diff
@@ -66,6 +54,19 @@ end
 ### Send an MMS
 
 ``` ruby
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
```

Comment:
Think we can probably drop this line, since you got rid of it above as well. Ruby 1.8.7, the last time you had to specifically require rubygems, is so out of date it's not even funny any more.